.github/workflows/rubocop.yml

```diff
@@ -0,0 +1,19 @@
+name: RuboCop
```

Comment:
Looks like you need to install some native libs.

Reply:
Yeah, I didn't disable the groups properly. I wanted to end-up with as minimal gem set as possible.